step/build_id_getter.go

```diff
@@ -68,6 +71,10 @@ func (bg BuildIDGetter) createKeyValuePairSlice() []keyValuePair {
 	var stageWorkflowMap []keyValuePair
 	for _, stage := range bg.FinishedStages {
 		for _, wf := range stage.Workflows {
+			if wf.ExternalId == "" {
+				bg.logger.Printf("Empty external ID for workflow %s in stage %s", wf.Name, stage.Name)
```

Comment:
I don't think customers will/should know anything about external ID. Can we log something like this:
`Skipping workflow %s in stage %s. Workflow was not executed.`?
Also, I would wait a confirmation from a mobile engineer to make sure this is the right way to log information for the customer.

Reply:
@godrei can you please confirm the logging function?